mtenn/tests/test_model_config.py

```diff
@@ -65,3 +66,58 @@ def test_random_seed_schnet():
         for p1, p2 in zip(set_model1.parameters(), set_model2.parameters())
     ]
     assert sum(set_equal) == len(set_equal)
+
+@pytest.mark.skipif(not HAS_VISNET, reason="requires VisNet from nightly PyG")
+def test_random_seed_visnet():
+    rand_config = ViSNetModelConfig()
+    set_config = ViSNetModelConfig(rand_seed=10)
+
+    rand_model1 = rand_config.build()
+    rand_model2 = rand_config.build()
+    set_model1 = set_config.build()
+    set_model2 = set_config.build()
+
+    rand_equal = [
+        (p1 == p2).all()
+        for p1, p2 in zip(rand_model1.parameters(), rand_model2.parameters())
+    ]
+    assert sum(rand_equal) < len(rand_equal)
+
+    set_equal = [
+        (p1 == p2).all()
+        for p1, p2 in zip(set_model1.parameters(), set_model2.parameters())
+    ]
+    assert sum(set_equal) == len(set_equal)
+
+@pytest.mark.skipif(not HAS_VISNET, reason="requires VisNet from nightly PyG")
+def test_random_seed_visnet_from_pyg():
```

Comment:
we're not really testing the `random_seed` part of this so we should probably change the name
```suggestion
def test_visnet_from_pyg():
```